api/api-service.js

```diff
@@ -9,20 +9,20 @@ var Seneca = require('seneca')
 
 var server = new Hapi.Server()
 
-server.connection({ 
+server.connection({
   port: PORT
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
       legacy: { logging: false },
```

Comment:
I'll fix this in the other PR about upgrading seneca 3.
Two questions: 
1- what was it for before?
2- we may want to fix logging examples: https://github.com/senecajs/seneca/blob/master/docs/examples/custom-logger.js#L34